packages/format-message/README.md

```diff
@@ -260,6 +260,27 @@ produces the same tree as
 </div>
 ```
 
+You can also provide an Object and use tag names that match the object's property names
```

Comment:
As you pointed out, there's probably not a great reason to have a restriction on valid tag names. If someone wants to have `<{[?>hello</{[?>` than presumably they have a good reason. ;)